src/toolkits/object_detection/od_darknet_yolo_model_trainer.cpp

```diff
@@ -252,6 +260,7 @@ EncodedInputBatch EncodeDarknetYOLO(InputBatch input_batch,
   result.iteration_id = input_batch.iteration_id;
   result.images = std::move(input_batch.images);
   result.annotations = std::move(input_batch.annotations);
+  result.image_sizes = std::move(input_batch.image_sizes);
```

Comment:
What is this for?

Reply:
Good question. For prediction, we want to yield predicted bounding boxes in image coordinates. Without this, we don't have a convenient way to scale our (normalized) bounding box predictions.

Reply:
Makes sense